GoodMorning/GoodMorning/CheckBoxButton.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CheckBoxButton.swift
+//  GoodMorning
+//
+//  Created by Miro on 2023/08/04.
+//
+
+import UIKit
+
+class CheckBoxButton: UIButton {
+
+    var isChecked = false {
+        didSet {
+            self.updateCheckMarkImage()
+        }
+    }
+
+    weak var delegate: CheckBoxButtonDelegate?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureAttributes()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func configureAttributes() {
+        backgroundColor = .white
+        layer.cornerRadius = 10
+        layer.borderColor = UIColor.gray.cgColor
+        layer.borderWidth = 1
+        addTarget(self, action: #selector(didButtonTapped), for: .touchUpInside)
+        updateCheckMarkImage()
+    }
+
+    @objc private  func didButtonTapped() {
+        isChecked.toggle()
+    }
```

Comment:
저희가 그린 Figma를 보면 체크박스가 선택 되었을 때  background가 네이비 색상으로 되어있어요. 
이 부분 구현을 여기서 하면 어떨까 해요!

```suggestion
        @objc private  func didButtonTapped() {
        isSelected.toggle()
        backgroundColor = isSelected ? .design(.checkBox) : .white
    }
```

Reply:
Figma대로하면 위와 같이 하면 좋을 거 같아요! 근데, 막상해보니까 background는 그냥 white인게 조금 더 깔끔해보이는 거 같기도해서요🥲 혹시 써니는 뭐가 더 UI적으로 괜찮다고 생각하시나요?